.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "3.6"
+cache: pip
+install:
+  - pip install -r requirements.txt
+script:
+  - python ./init.py

```

Comment:
@Mansi145 The script is used to check if changes from the most recent build pass all tests. This would change to running the unit tests for the API modules and the linter as well. For reference, see [this](https://github.com/janakrajchadha/web-monitoring-processing/blob/master/circle.yml). Please note that this reference is based on circle CI, and there might be a minor change in the syntax for travis CI.

Additionally, the call to the current unit tests file would do nothing since there is no call within the file to actually run those tests. So, a minor change will have to be made for that as well. [Here's](https://docs.pytest.org/en/latest/usage.html#calling-pytest-from-python-code) a link for reference.